tests/system/RESTful/ResourcePresenterTest.php

```diff
@@ -59,6 +59,7 @@ protected function setUp(): void
 
         $config            = new App();
         $this->codeigniter = new MockCodeIgniter($config);
+        $this->codeigniter->setContext('web');
```

Comment:
I think these are no longer needed with the default property?

Reply:
Removed.